development/dev.env

```diff
@@ -52,3 +52,6 @@ NAUTOBOT_SUPERUSER_API_TOKEN=0123456789abcdef0123456789abcdef01234567
 
 NAUTOBOT_INSTALLATION_METRICS_ENABLED=false
 NAUTOBOT_LOG_DEPRECATION_WARNINGS=true
+
+K8S_AUTH_TOKEN="change-me"
```

Comment:
When does this need to be changed in the dev environment?

Reply:
Removed as not necessary